File: C# Basics/Conditions/SwimmingRecord.cs
using System;

namespace SwimmingREcord
{
    class Program
    {
        static void Main(string[] args)
        {
            double currentRecord = double.Parse(Console.ReadLine());
            double distance = double.Parse(Console.ReadLine());
            double oneMemerScore = double.Parse(Console.ReadLine());

            double distanceScore = oneMemerScore * distance;
            double delay = Math.Floor(distance / 15) * 12.5;
            double finalScore = distanceScore + delay;

            if (finalScore >= currentRecord)
            {
                Console.WriteLine($"No, he failed! He was {(finalScore-currentRecord):f2} seconds slower.");
            }
            else
            {
                Console.WriteLine($"Yes, he succeeded! The new world record is {finalScore:f2} seconds.");
            }
        }
    }
}

File: Java Basics/ConditionsEx/SwimmingRecord.java
package ConditionsEx;

import java.util.Scanner;

public class SwimmingRecord {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double currentRecord = Double.parseDouble(scanner.nextLine());
        double distance = Double.parseDouble(scanner.nextLine());
        double timePerMeter = Double.parseDouble(scanner.nextLine());

        double timeToFinish = timePerMeter * distance;
        int delay = (int)distance / 15;
        timeToFinish += delay * 12.5;

        if (timeToFinish < currentRecord) {
            System.out.printf("Yes, he succeeded! The new world record is %.2f seconds.", timeToFinish);
        } else {
            System.out.printf("No, he failed! He was %.2f seconds slower.", timeToFinish - currentRecord);
        }
    }
}
